en/02_Developer_Guides/00_Model/04_Data_Types_and_Casting.md

```diff
@@ -22,6 +22,7 @@ use SilverStripe\ORM\DataObject;
 
 class Player extends DataObject
 {
+    // ...
```

Comment:
Adding these to indicate we're skipping some best practices (e.g. `table_name`) for the sake of brevity.